app/controllers/api/v1/merchants/random_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::Merchants::SearchController < ApplicationController
```

Comment:
This file is named `random_controller` and the class is final class is `SearchController` Not sure if that matters grade wise

Reply:
Good catch @mikeyduece! This has been fixed.